### ISSUE TITLE
Multi-file debugging with breakpoints

### ISSUE THREAD

**Issue by MikeShepard** [PowerShell/vscode-powershell#1898]
Issue Type: <b>Bug</b>

Script1.ps1:
.\script2
#what is going on?
#I'm not sure any more
#please just stop!
#how 'bout now?

Script2.ps1:
.\script2
#what is going on?
#I'm not sure any more
#please just stop!
#how 'bout now?

Breakpoints on first line of both scripts, start debugging script1.  When you step into the call to script2, the cursor jumps down to line 5 of script1.  If you do "list" in the debug console, PowerShell tells you it's currently on line 5 of script2.   I have reproduced this with the latest PowerShell and PowerShell preview extensions.
![VSCodeDebugging](https://user-images.githubusercontent.com/5541154/56297517-13af7d00-60f6-11e9-820c-54b3505de08d.gif)

Extension version: 1.12.1
VS Code version: Code 1.33.1 (51b0b28134d51361cf996d2f0a1c698247aeabd8, 2019-04-11T08:27:14.102Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6600U CPU @ 2.60GHz (4 x 2808)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.88GB (8.14GB free)|
|Process Argv|--file-uri file:///c%3A/temp/JHB_Account_Review/jhbaccountReview.code-workspace|
|Screen Reader|no|
|VM|0%|

</details>

**Comment by SydneyhSmith** [comment]
@MikeShepard thanks for reporting this, it would be really helpful if you could also attach the logs--instructions for how to do that are [here ](https://github.com/PowerShell/vscode-powershell/blob/master/docs/troubleshooting.md#providing-information-about-your-environment)

**Comment by SydneyhSmith** [comment]
Does this repro when you use the .ps1 extension into your code i.e. 
```
.\script2.ps1
#what is going on?
#I'm not sure any more
#please just stop!
#how 'bout now?
```

**Comment by MikeShepard** [comment]
Yes, adding the .ps1 extension in the first line of the first file doesn't change the behavior (it still misbehaves).
[1555625351-7b3942e8-c29e-4768-9ef3-6d82c8ffc0ae1555625252197.zip](https://github.com/PowerShell/vscode-powershell/files/3096325/1555625351-7b3942e8-c29e-4768-9ef3-6d82c8ffc0ae1555625252197.zip)

**Comment by MikeShepard** [comment]
I've reproduced this on Linux (Ubuntu):
[logs.zip](https://github.com/PowerShell/vscode-powershell/files/3100317/logs.zip)

**Comment by MikeShepard** [comment]
I've also reproduced this in PS Core on Windows
[Logs.zip](https://github.com/PowerShell/vscode-powershell/files/3100324/Logs.zip)

FWIW, the 3 sets of log files are from 3 different machines, all up-to-date on VS Code and the extension.
